deploy-cockroachdb-on-google-cloud-platform.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Deploy CockroachDB on Google Cloud Platform GCE
+summary: Learn how to deploy CockroachDB on Google Cloud Platform's Compute Engine.
+toc: false
+toc_not_nested: true
+---
+
+This page shows you how to manually deploy an insecure multi-node CockroachDB cluster on Google Cloud Platform's Compute Engine.
+
+{{site.data.alerts.callout_warning}} This guide demonstrates how to deploy an <strong>insecure</strong> cluster, which we do not recommend for data in production. We'll update this page once we've improved the workflow for deploying secure clusters.{{site.data.alerts.end}}
```

Comment:
Can you explain what plans we have for improving the workflow for deploying secure clusters? I ask since it's also relevant for deploying on top of container orchestration systems like Kubernetes.